src/components/Voting/Referendum/Items/ReferendumItem/index.js

```diff
@@ -26,16 +50,67 @@ const alreadyVotedButton = (alreadyVoted) => (alreadyVoted === 'Aye'
     </Button>
   ));
 
+const voteButtonsContainer = (alreadyVoted, delegating, buttonVoteCallback, referendumData) => {
+  if (alreadyVoted) { return alreadyVotedButton(alreadyVoted); }
+  if (delegating) { return 'Undelegate to vote individually'; }
+  return voteButtons(buttonVoteCallback, referendumData);
+};
+
+const blacklistButton = (proposalHash, referendumIndex) => {
+  const dispatch = useDispatch();
+  const members = useSelector(congressSelectors.members);
+  const user = useSelector(blockchainSelectors.userWalletAddressSelector);
+
+  useEffect(() => {
+    dispatch(congressActions.getMembers.call());
+  }, [dispatch]);
+
+  const userIsMember = members.map((m) => m.toString()).includes(user);
+
+  if (!userIsMember) return null;
+
+  const blacklistMotion = () => dispatch(
+    congressActions.createBlacklistMotion.call({
+      proposalHash,
+      referendumIndex,
+    }),
+  );
+
+  return (
+    <Button small secondary onClick={() => { blacklistMotion(); }}>
+      Cancel
+    </Button>
+  );
+};
+
 function ReferendumItem({
-  name, createdBy, externalLink, description, yayVotes, nayVotes, hash, alreadyVoted, buttonVoteCallback, votingTimeLeft, referendumIndex, delegating,
+  name,
+  createdBy,
+  externalLink,
+  description,
+  yayVotes,
+  nayVotes,
+  hash,
+  alreadyVoted,
+  buttonVoteCallback,
+  votingTimeLeft,
+  referendumIndex,
+  delegating,
 }) {
-  const progressBarRatio = yayVotes > 0 ? `${(formatPolkadotBalance(yayVotes)) / (formatPolkadotBalance(yayVotes) + formatPolkadotBalance(nayVotes)) * 100}%` : '0%';
+  const progressBarRatio = yayVotes > 0
+    ? `${(formatPolkadotBalance(yayVotes)
+      / (formatPolkadotBalance(yayVotes) + formatPolkadotBalance(nayVotes)))
+       * 100}%`
+    : '0%';
   return (
     <Card
       title={name}
       className={styles.referendumItemContainer}
     >
       <div>
+        <div className={styles.rowEnd}>
+          {blacklistButton(hash, referendumIndex)}
```

Comment:
This should be `<BlacklistButton hash={hash} referendumIndex={referendumIndex} />` or we risk running into hook call order issues.